### PR TITLE
Adding 2 projects for the game Lego Universe

### PR DESCRIPTION
**Pull request by MajickTek**
This pull request adds the 2 largest open-source projects emulating [Lego Universe](https://en.wikipedia.org/wiki/Lego_Universe), a NetDevil MMORPG released in 2010.

These 2 projects have decently-sized communities and both are almost 100% functionally equivalent to the original game.

### CODE REVIEW

**Comment by MajickTek** [comment]
Hmm the bot says format issues exist and the youtube link is not recognized as valid.

If somebody could help me re-format the YAML that would be greatly appreciated.

File: games/d.yaml [hunk 1]
```diff
@@ -1197,3 +1197,28 @@
...
+  - C++
+  license:
+  - AGPL3
+  info: This project is a server emulating an MMO developed by NetDevil and released in 2010.
+  updated: '2022-02-15'
+  images:
+  - 'https://pbs.twimg.com/media/E5jL9BvWQAQOeYQ?format=jpg&name=900x900'
+  - 'https://pbs.twimg.com/media/E17cXShWYAMLJI9?format=jpg&name=large'
+  video:
+  youtube: ZhzMy-o0ByM
```
Comment (cxong) [inline games/d.yaml:1224]:
```suggestion
    youtube: ZhzMy-o0ByM
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Review by cxong — CHANGES_REQUESTED** [review]
need to add original game Lego Universe

File: games/d.yaml [hunk 2]
```diff
@@ -1197,3 +1197,28 @@
   updated: 2019-11-14
   images:
   - https://raw.githubusercontent.com/akien-mga/dynadungeons/master/background.png
+  
+- name: Darkflame Universe
+  originals:
+  - Lego Universe
+  type: remake
+  repo: 'https://github.com/DarkflameServer/DarkflameUniverse'
```
Comment (cxong) [inline games/d.yaml:1205]:
```suggestion
  repo: https://github.com/DarkflameUniverse/DarkflameServer
```

File: games/u.yaml [hunk 3]
```diff
@@ -479,3 +479,22 @@
...
+  - Local
+  - Online
+  - LAN
+  content: commercial
+  lang:
+  - 'C#'
+  license:
+  - GPL3
+  info: A Server project emulating original data from NetDevil's Lego Universe (2010)
+  updated: '2022-02-15'
```
Comment (cxong) [inline games/u.yaml:500]:
```suggestion
  updated: '2022-02-15'
  video:
    youtube: _b6HYmdcA8c
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Pull request merged**